superset-frontend/src/components/OmniContainer/OmniContainer.test.tsx

```diff
@@ -84,40 +84,6 @@ test('Open Omnibar with ctrl + k with featureflag enabled', () => {
   ).not.toBeVisible();
 });
 
-test('Open Omnibar with ctrl + s with featureflag enabled', () => {
```

Comment:
It seems like this test could be written with `K` and `esc` keystrokes, to keep it rather than delete it.